static-assets/scripts/main.js

```diff
@@ -1456,6 +1465,19 @@
     }
   ]);
 
+  app.controller('TokenManagementCtrl', [
+    '$scope',
+    function($scope) {
+      const workarea = document.querySelector('#token-management-view');
+      const el = document.createElement('div');
+
+      $(workarea).html('');
```

Comment:
Seems pointless. The container is empty as it comes from the template.
Does angular keep around the view as it unmounts? If that's the case, we should be unmounting the react component too.